core/src/test/scala/integration/kafka/admin/TopicCommandIntegrationTest.scala

```diff
@@ -172,13 +172,11 @@ class TopicCommandIntegrationTest extends KafkaServerTestHarness with Logging wi
   @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
   @ValueSource(strings = Array("zk", "kraft"))
   def testCreateWhenAlreadyExists(quorum: String): Unit = {
-    val numPartitions = 1
-
     // create the topic
+    TestUtils.createTopicWithAdmin(adminClient, testTopicName, brokers, 1, 1)
+
     val createOpts = new TopicCommandOptions(
       Array("--partitions", numPartitions.toString, "--replication-factor", "1", "--topic", testTopicName))
-    createAndWaitTopic(createOpts)
```

Comment:
Oh, I specified incorrect details in JIRA! We don't want to replace `createAndWaitTopic` because that is the function which is under test in this file. We want to replace occurrences of `adminClient.createTopics` with the utility function. Apologies for the confusion. Can you please revert back the changes to createAndWaitTopic (but keep the ones where we are using adminClient.createTopics.

Reply:
I'm sorry, I misunderstood your meaning in JIRA. I have already reverted back the changes to createAndWaitTopic.